cmd/crc/cmd/config/config.go

```diff
@@ -24,6 +24,7 @@ const (
 	HTTPSProxy           = "https-proxy"
 	NoProxy              = "no-proxy"
 	ProxyCAFile          = "proxy-ca-file"
+	EnableTelemetry      = "enable-telemetry"
```

Comment:
we have to make sure this gets a description, as it is suggestive of something the cluster could do

Reply:
@gbraad should we name it `EnableCRCTelemetry` ?

Reply:
I think it is more about `"enable-telemetry"`...